app/views/apps/index.html.haml

```diff
@@ -1,55 +1,84 @@
+!!!
+%html
+  %head
+    = stylesheet_link_tag "apps"
+
 .page-header
   %h1
     Listing Apps
-    = link_to 'New App', new_app_path, :class => 'btn btn-success pull-right'
+    = link_to new_app_path, :class => 'btn btn-success pull-right m-3' do 
+      New App
+      %i.fas.fa-pencil-alt
   .help-block Click an App name to see more information
 
-#filters
-  %label.checkbox-inline
-    %div#deploy
-      %strong Deployment Status:
-      %em #{" (" + @total_deploy.to_s + ")"}
-      %br
-      - App.getAllDeploymentStatuses.each do |status|
-        = check_box_tag status, 1, true, :class => 'filter'
-        %span{:class => status}= status.to_s.humanize
-        %em (#{@deployment_map[status.to_s].to_i})
+  - content_for :filters do
+    - $filters = ""
+    = form_tag "/", :method => :get do
+      %div#deploy
+        %strong Deployment Status:
+        %em #{" (" + @total_deploy.to_s + ")"}
         %br
```

Comment:
For the changes below, they are pagination for the App index page. Since they are written in the same file with UI, we have to do the PR as a whole.